src/Seq.App.Opsgenie/Seq.App.Opsgenie.csproj

```diff
@@ -12,6 +12,13 @@
         <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
         <TreatSpecificWarningsAsErrors />
         <PackageLicenseFile>LICENSE</PackageLicenseFile>
+        <AssemblyName>Seq.App.Opsgenie2</AssemblyName>
+        <Version>1.1.18</Version>
+        <AssemblyVersion>1.1.18.0</AssemblyVersion>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Shouldn't need this using the standard `/build.ps1`

Reply:
Yes - was only used for my purposes